wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/ProxyCommand.java

```diff
@@ -45,7 +51,7 @@ public ProxyCommand(Supplier<Command> supplier) {
    */
   @SuppressWarnings("this-escape")
   public ProxyCommand(Command command) {
-    this(() -> command);
+    m_supplier = () -> requireNonNullParam(command, "command", "ProxyCommand");
```

Comment:
The null check shouldn't be inside the supplier; it should validate the argument on constructor invocation.